modules/uploading-model-files-to-pvc.adoc

```diff
@@ -0,0 +1,53 @@
+:_module-type: PROCEDURE
+
+[id="uploading-model-files-to-pvc_{context}"]
+
+= Uploading model files to a Persistent Volume Claim (PVC)
+
+When deploying a model, you can serve it from a preexisting Persistent Volume Claim (PVC) where your model files are stored. You can upload your local model files to a PVC by using a running workbench.
```

Comment:
suggestion instead of "by using a running workbench":

You can upload your local model files to a PVC in the IDE that you access from a running workbench.